script/foreman-debug

```diff
@@ -39,20 +39,20 @@ Please note that the rsync transmission is UNENCRYPTED.
 UPLOADUSAGE
 }
 
-# filter for patterns like password=XYZ or secret: abc!@#$123
+# Filter for patterns like password=XYZ, -storepass XYZ or secret: XYZ
 FILTER_WORDS=(
 password
 PASSWORD
 default_password
+key
```

Comment:
What strings are you talking about specifically? The ticket mentions "oauth_consumer_key" - if that's the only one (and I'm unsure it's a secret, as there is a separate secret), then it's probably worth naming.